security-privacy-questionnaire.md

```diff
@@ -0,0 +1,82 @@
+# Security and Privacy Questionnaire
+
+This document answers the [W3C Security and Privacy
+Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/) for the
+Picture-in-Picture specification.
+
+**Does this specification deal with personally-identifiable information?**
+
+No.
+
+**Does this specification deal with high-value data?**
+
+No.
+
+**Does this specification introduce new state for an origin that persists across
+browsing sessions?**
+
+No.
+
+**Does this specification expose persistent, cross-origin state to the web?**
+
+No.
+
+**Does this specification expose any other data to an origin that it doesn’t
+currently have access to?**
+
+No.
+
+**Does this specification enable new script execution/loading mechanisms?**
+
+No.
+
+**Does this specification allow an origin access to a user’s location?**
+
+No.
+
+**Does this specification allow an origin access to sensors on a user’s
+device?**
+
+No.
+
+**Does this specification allow an origin access to aspects of a user’s local
+computing environment?**
+
+Yes. It allows an origin to know if Picture-in-Picture is enabled 
+on user's local computing environment.
+
+**Does this specification allow an origin access to other devices?**
+
+No.
+
+**Does this specification allow an origin some measure of control over a user
+agent’s native UI?**
+
+No.
```

Comment:
I would actually say "Yes" to that one :)

Reply:
Done.